components/automate-ui/src/app/modules/policy/details/policy-details.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="app-main-container">
+  <app-settings-sidebar sidebar></app-settings-sidebar>
+
+  <div class="container">
+    <chef-notifications></chef-notifications>
```

Comment:
Note to self (and others). This file is largely unchanged, just renamed, like its brethren. The only changes are adding L1, L2, L5, and the closing "bracket" to L1 on L81.